prebuild/index.js

```diff
@@ -55,25 +59,90 @@ function prebuildify () {
 }
 
 function prebuildTarget (arch, target) {
+  if (NAPI_RS === 'true' && platform === 'linux' && arch === 'ia32') return
```

Comment:
I spent a while debugging an issue with napi-rs building for `linux-ia32` where the `napi build` command couldn't find cargo, despite all rust packages [added to path beforehand](https://github.com/DataDog/action-prebuildify/blob/07a63d9da64b8a8161a60e6c0d5bb0306281e8d7/prebuild/index.js#L116). All other arch/platforms don't have this problem